src/course/syllabus/apprenticeship/server-side-app/schedule.md

```diff
@@ -2,13 +2,28 @@
 layout: schedule
 schedule:
   monday:
-    - name: Employer day
+    - name: Validation workshop
```

Comment:
I think it'd be helpful to set the time expectations here. The workshop only needs ~2 hours, the rest of the day can be given to revision

Reply:
Sure, thanks 👍. Was going to clarify verbally as aware they don't all keep the hours on Monday, but you're right - it is still helpful for setting expectations re broad time commitment